### PR TITLE
update create explore group view

### PR DESCRIPTION
**Pull request by Elyzsium**
**Description of PR**

create explore group view

---

Please check all that apply.

#### Checklist:

- [x] Rebased with master
- [x] PR Description provided
- [ ] Did you communicate these changes with the affected teams?
- [ ] Did unit tests/integration tests/E2E tests get properly added/updated?
- [ ] Is this change production-ready including testing, feature toggles, configurations, and deployment considerations?

#### Frontend Checklist:

- [x] Is this responsive and mobile friendly?
- [ ] Did you follow the [ARIA specs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA)

#### Backend Checklist:

- [ ] Does the endpoint return the correct success/error format?

---

**Screenshots**  

group view2

---

**Test Coverage**  
_add relevant screenshots here_

### CODE REVIEW

**Comment by KristofaJosh** [comment]
Task linked: [CU-86bwt2ybt Create explore group view](https://app.clickup.com/t/86bwt2ybt)

**Review by pelzman — COMMENTED** [review]
image

you can make the table a component and pass the necessary parameters, this is because the table would still be re-use on another card, just to prevent repeating yourself

for styling
styling mobile view first would have saved you some stress and reduced your lines of code..

the image got too small on the small screen , adjust

**Review by pelzman — COMMENTED** [review]
Note: if you are doing mobile first, your width would be 100%

and besides… this your css is a bit long , it needs refactoring

**Review by pelzman — COMMENTED** [review]
image

for the table - try and use divs instead of the table tag
read up on grid-template-area on https://www.youtube.com/watch?v=Ny7LkAt4btI&t=513s.   and also check the way i implemented the table on transacton page as a guide.

File: apps/client/src/pages/Dashboard/Groups/group/exploreGroupView/index.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
+import { Text } from 'components'
+import styles from './exploreGroup.module.scss'
+import frame from './images/Frame.svg'
+import rightless from './images/rightless.svg'
+import TableComponent from '../table/TableComponent'
+// import TableComponent from '../table/TableComponent'
+
+const cardData = [
```
Comment (Dallat65) [inline apps/client/src/pages/Dashboard/Groups/group/exploreGroupView/index.tsx:8]: Good job Rukky but pls note data wont come like this from the backend. Pls sync with backend guys.

File: apps/client/src/pages/Dashboard/Groups/group/exploreGroupView/index.tsx [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                />
+                <Text
+                  content={card.value2}
+                  size={'Small'}
+                  className={styles.rightOne}
+                />
+              </div>
+            </div>
+          ))}
+        </div> */}
```
Comment (KristofaJosh) [inline apps/client/src/pages/Dashboard/Groups/group/exploreGroupView/index.tsx:125]: needed?

**Comment by KristofaJosh** [comment]
@pelzman using the table in this situation is fine.

although, 
- I have issues with the top section of the mobile view, it looks undone.

image


- lastly, your breadcrumb is not interactive.

**Review by KristofaJosh — CHANGES_REQUESTED** [review]
Make research on breadcrumbs

**Review by KristofaJosh — DISMISSED** [review]

**Pull request closed without merging**